src/theming.jl

```diff
@@ -89,7 +89,8 @@ const MAKIE_DEFAULT_THEME = Attributes(
         pt_per_unit = 0.75,
         antialias = :best,
         visible = true,
-        start_renderloop = false
+        start_renderloop = false,
+        pdf_version = "1.7"
```

Comment:
Would this cause a change compared to current output?

Reply:
It is my understanding that cairo sets the PDF version to 1.7 by default (see here https://gitlab.freedesktop.org/cairo/cairo/-/blob/master/src/cairo-pdf-surface.c?ref_type=heads#L518), so I believe that "restricting" the version to 1.7 is the same as not restricting at all.